tests/integration/ethernet_mtu_test.py

```diff
@@ -135,6 +138,13 @@ def test_decrease_to_lower_than_min_ipv6_iface_mtu(eth1_with_ipv6):
     assertlib.assert_state(original_state)
 
 
+@pytest.mark.parametrize("mtu", [MINIMAL_IPV6_MTU - 1, 926, 576])
```

Comment:
What are these `926, 576` mtu values represent?
These are not unit tests, why do we need to test these?

Reply:
After discuss this on IRC. We are only testing 576. Thanks!